src/js/player.js

```diff
@@ -403,6 +404,7 @@ class MediaElementPlayer {
 	}
 
 	showControls (doAnimation) {
+
```

Comment:
Functions should not have an empty line at the start. Please correct for all the functions, where you changed this.

Reply:
Thanks